### ISSUE TITLE
Getting errors while applying patch

### ISSUE THREAD

**Issue by manikantagamer** [NVIDIAGameWorks/RTXGI-DDGI#45]
![issue](https://user-images.githubusercontent.com/12003219/111626081-97f36900-8813-11eb-9207-5bd12dae3eb4.PNG)

Hi can anyone help me to fix the issues, refer the image attached image

**Comment by acmarrs-nvidia** [comment]
This is a duplicate of #35.

How to properly clone UE4, the RTXGI SDK, and apply the patch is covered in the [UE4 Getting Started Guide](https://github.com/NVIDIAGameWorks/RTXGI/wiki/UE4-Plugin-Getting-Started-Guide).

**Comment by manikantagamer** [comment]
Hi acmarrs,
I have replicated the process mentioned in the starter guide, when I do {git apply --check RTXGI-NvRTX4.26.1}, it returned with no issues, refer to screenshot.
![Check](https://user-images.githubusercontent.com/12003219/111899549-54873d80-8a53-11eb-8dfd-210f279545e4.PNG)

but when I tried installing the plugin with {git apply RTXGI-NvRTX4.26.1}, it throwed up lot of errors.
even though there are issues, plugin got installed to Plugins/Runtime/Nvidea, it would great if you guys work on a binary version to install this plugin.

![Install](https://user-images.githubusercontent.com/12003219/111899552-56e99780-8a53-11eb-9057-77ea0618bf49.PNG)
